hadoop-ozone/dist/src/main/k8s/examples/ozone/test.sh

```diff
@@ -28,7 +28,19 @@ regenerate_resources
 
 start_k8s_env
 
-execute_robot_test scm-0 smoketest/basic/basic.robot
```

Comment:
Don't we still want to run this test, in addition to those below?

Reply:
@adoroszlai The change for this file is by cherry picking of your commits. Could you pls let me know why you delete this line before? If we need it back, I can simply put it back. Just want to know your consideration. Thanks

Reply:
I'm OK either way.

`basic.robot` has two tests:

 * HTTP request for static web resource
 * Freon key generation/validation

The latter (Freon) is performed by the new code, too, so only the web test is missing.